### PR TITLE
[Naitei] ft:Search_for_Guest

### PR DESCRIPTION
**Pull request by ThaiKhacDuoc**
## Related Tickets
- ticket redmine

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)

image

https://github.com/user-attachments/assets/f836a441-13d5-4a05-b554-b8368b3fd421

### CODE REVIEW

**Comment by ThaiKhacDuoc** [comment]
ready

File: config/locales/en.yml [hunk 1]
```diff
@@ -172,7 +173,6 @@ en:
     user_not_correct: "User not correct"
     user_not_admin: "You are not admin"
   admin:
-    dashboard: "Admin Dashboard"
```
Comment (ghost) [inline config/locales/en.yml]: không xóa ở tiếng việt à em

File: app/models/product.rb [hunk 2]
```diff
@@ -11,9 +11,9 @@ class Product < ApplicationRecord
   delegate :name, to: :category, prefix: true
   scope :order_by_comment, ->{order("COUNT(comments.id) DESC")}
   scope :product_by_comments, (lambda do
-    joins(:comments)
-      .group("products.id")
-  end)
+                                 joins(:comments)
+                                   .group("products.id")
+                               end)
```
Comment (ghost) [inline app/models/product.rb]: Sao chỗ này lại thụt vào nhiều thế

**Review by ghost — COMMENTED** [review]
cmt

**Comment by ThaiKhacDuoc** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**